cypress/integration/2-settings/stake-abpt.ts

```diff
@@ -0,0 +1,26 @@
+import { configEnvWithTenderlyMainnetFork } from '../../support/steps/configuration.steps';
+
+describe('Checking ABPT token modal on Staking page', () => {
+  describe('CASE1:Open Stake page', () => {
+    configEnvWithTenderlyMainnetFork({});
+
+    it('step1:Open Stake page', () => {
+      cy.get('a[href*="/staking/"]').click();
+    });
+  });
+
+  describe('CASE2:GET ABP Token', () => {
+    it('step2:Get ABP Token', () => {
+      cy.contains('Stake ABPT').click();
+      cy.contains('Get ABP Token').click();
+    });
+
+    it('step3:Verify does Get ABP Token modal is opened', () => {
+      cy.contains('Go to Balancer Pool').should(
```

Comment:
Same thing, could do
`cy.get('[data-cy="stakeBtn_ABPT"]').click();`

This one would probably be more likely to break. Unlikely that we'd change the link name, but in this case if the button text we're to change, the test would fail :)